tests/test-policies/src/test/java/org/mule/test/policy/PolicyTestCase.java

```diff
@@ -97,6 +115,13 @@ public void policyDisposalStopsSchedulers() {
     assertThat(schedulerService.getSchedulers().size(), is(0));
   }
 
+  @Test
+  public void policyCacheEntryGetsEvictedOnFlowDisposal() throws Exception {
+    Thread.sleep(2000);
```

Comment:
if you need to wait for the flow to have kicked in some times, use a latch instead of sleep